docs/proposals/lifelong-learning.md

```diff
@@ -80,7 +80,7 @@ Here is a list of validations we need to support :
 1. The edgenode name specified in the crd should exist in k8s.
 
 ### Lifelong learning job sample
-See the [source](/build/crd-samples/sedna/Lifelonglearningjob_v1alpha1.yaml) for an example.
+See the [source](/build/crds-samples/sedna/lifelonglearningjob_v1alpha1.yaml) for an example.
```

Comment:
better to self-check the link in browser.
```suggestion
See the [source](/build/crd-samples/sedna/lifelonglearningjob_v1alpha1.yaml) for an example.
```

Reply:
fix